calibr8/utils.py

```diff
@@ -172,22 +133,24 @@ def plot_t_band(ax, independent, mu, scale, df, *, residual_type: typing.Optiona
     artists : list of matplotlib.Artist
         the created artists (1x Line2D, 6x PolyCollection (alternating plot & legend))
     """
+    assert hasattr(model.scipy_dist, "ppf"), "Only Scipy distributions with a ppf method can be used for the continuous likelihood bands."
+    params =  list(model.predict_dependent(independent))
     if residual_type:
         artists = ax.plot(independent, numpy.repeat(0, len(independent)), color='green')
     else:
-        artists = ax.plot(independent, mu, color='green')
+        artists = ax.plot(independent, params[0], color='green')
```

Comment:
I think we could do something like `model.scipy_dist.ppf(0.5, **model.to_scipy(*params))` to get the median.

...and store that as a `median` variable so the `params[0]` below can use it too

Reply:
I think the exponential or beta distributions are good examples(?)

Reply:
I think the part with the residuals might be tricky in that case (the code below where I replaced the first argument of params by 0 for those). I guess that won't work for a beta distribution...

Reply:
@m.osthege I made a try here, please have a look if you think this might work.